controllers/stargate/stargate_controller.go

```diff
@@ -398,42 +400,39 @@ func (r *StargateReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 func (r *StargateReconciler) reconcileStargateConfigMap(
 	ctx context.Context,
 	stargateObject *api.Stargate,
-	desiredConfig map[string]interface{},
-	userConfigMapContent string,
+	dcConfig map[string]interface{},
+	userCassandraYaml, userCqlYaml string,
```

Comment:
A bit of naming/refactoring in this function, because it was getting really confusing with two files.